ktor-server/ktor-server-core/jvm/src/io/ktor/server/http/content/PreCompressed.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.http.content
+
+import io.ktor.http.*
+import io.ktor.http.content.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.util.*
+import java.io.*
+
+/**
+ * Supported pre compressed file types and associated extensions
+ *
+ * **See Also:** [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding)
+ */
+public enum class CompressedFileType(public val extension: String, public val encoding: String = extension) {
+    // https://www.theregister.co.uk/2015/10/11/googles_bro_file_format_changed_to_br_after_gender_politics_worries/
+    BROTLI("br"),
+    GZIP("gz", "gzip");
+
+    @Deprecated("This will become internal")
```

Comment:
Should we have a replacement or link to the docs?

Reply:
No, it's used only internally and there is no need for it to be public